Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Fixes for Сartography Inaccuracies

Resolves #342.

_Locations in italics_ can be inserted into the 'Find Title' dialogue to find locations quicker.


## Changelog:
- Complete rework of the _Venomweb_ Vale in Tirisfal Glades.
- Added trees to the following:
  - Eastern Lordaeron (Western and Eastern Plaguelands)
  - Quel'thalas (Eversong Woods, Ghostlands, and Zul'Aman)
  - Ashenvale
  - Hyjal
  - Krarasang Wilds
  - _Paoquan_
- Trees removed from _Stoneplow_
- Reworked the _Blackwald_ region in Gilneas.
- _Gilneas_ City texture has been changed.
- The two rivers near _Sidosh_ have been redone.
- Azshara Fixes
  - The river west of _Ravencrest_ in Uldum has been removed.
  - The mountain range on the south edge of Azshara has been removed.
- Partial Darkshore Revamp
  - Rework of the Heightmap for the northern part of Darkshore. Includes a coastal shelf along with textured rock formations.
  - _Cliffspring_ River added.
- Added terrain for:
    - Plunder Isles and surrounding islands
    - Kul Tiras
    - Northrend
    - Zandalar
    - Broken Isles
    - Expedition Islands
    - Pandaria
- Removed trees in the water



## Known Issues:
@zumbak04 
- [ ] The provinces for Ahn'Qiraj need to be redrawn. 
  - A placeholder (ID #6379) has been placed in the newly added lands to the southwest of the region.
- [ ] Replace locators
- [ ] Several provinces from the old implementation of Uldum are now in Qiraji lands and need to be converted.
- [ ] Filter counties with one barony

<details>
<summary>Heightmap Overlay & WoW Cata Map</summary>

![](https://cdn.discordapp.com/attachments/751109648869752954/892495119482236938/unknown.png)
</details>

<details>
<summary>Province View</summary>

![image](https://user-images.githubusercontent.com/40177753/135155460-7a470de3-076d-4d9f-a16a-cf598c21546b.png)
</details>

## Planned Changes & Requests
- Northrend requires its own overhaul. The revisions outlined in #342 for Northrend will be added onto a new Northrend-only rework issue.
- Pandaria overhaul.
- A new tree object with yellow leaves for Quel'thalas and Quel'danas. @Warcraft-GoA-Development-Team/3d-artists 
<details>
<summary></summary>

![](https://wow.zamimg.com/uploads/screenshots/normal/287247-explore-isle-of-queldanas.jpg)
</details>




## Tests:


# How to test:
(This section shamelessly copied from @ercarp)

Consider the following when giving your feedback:

**Canonicity:** Verify the map changes are accurate to the information made available by official canon.

**Familiarity:** Observe whether or not the changes are recognizable and fitting for their respective locations.

**Aesthetics:** Comment on the general look and appeal on the changes. Suggestions should be as specific as possible.

Additionally, ensure that necessary gameplay features like Holdings are not obscured by map changes such as trees, water, or steep cliffs.

### -- @ValianBlue
